.hyperloop/checks/check-run-backend-suite.sh

```
@@ -0,0 +1,179 @@
+REPO_ROOT="$(git rev-parse --show-toplevel 2>/dev/null || true)"
+if [[ -z "$REPO_ROOT" ]]; then
+  echo "ERROR: Not inside a git repository — cannot determine repo root."
+  exit 1
+fi
+if [[ "$PWD" != "$REPO_ROOT" ]]; then
+  echo "⚠  NOTICE: Suite invoked from $PWD"
+  echo "   Normalizing CWD to repo root: $REPO_ROOT"
+  echo "   (Running from a subdirectory causes git pathspecs to silently"
+  echo "    match nothing — all content checks would produce false PASSes.)"
+  echo ""
+  cd "$REPO_ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)